reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md

```diff
@@ -21,161 +19,148 @@ to make the user experience a bit more familiar compared to the ISE.
 > and in the PowerShell extension since version 2020.3.0.
 
 The easiest way to replicate the ISE experience in Visual Studio Code is by turning on "ISE Mode".
-To do this, open the command pallet (<kbd>F1</kbd> OR <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> OR
-<kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> on macOS) and type in "ISE Mode".
-Select "PowerShell: Enable ISE Mode" from the list.
+To do this, open the command palette (<kbd>F1</kbd> OR <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>
+OR <kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> on macOS) and type in "ISE Mode". Select
+"PowerShell: Enable ISE Mode" from the list.
 
-This command will apply a lot of the settings found in this document automatically.
-The result looks like this:
+This command automatically applies the settings described below The result looks like this:
 
-![ISE Mode](media/How-To-Replicate-the-ISE-Experience-In-VSCode/3-ise-mode.png)
+![ISE Mode](media/How-To-Replicate-the-ISE-Experience-In-VS Code/3-ise-mode.png)
```

Comment:
This image is broken with the space in the URL.

Reply:
fixed